src/export/export_snapshot_s3_function.py

```diff
@@ -0,0 +1,52 @@
+import os
+import time
+
+import boto3
+
+import constants
+import utility as util
+
+
+def lambda_export_rds_snapshot_to_s3(event, context):
+    """start export task of RDS snapshot to S3 bucket"""
+    region = os.environ['Region']
+    rds = boto3.client('rds', region)
+    result = {}
+    instance_id = event['identifier']
+    epoch = int(time.time())
+    export_id = instance_id + "-" + str(epoch)
+    snapshot_id = instance_id + constants.SNAPSHOT_POSTFIX
+    snapshot_arn = get_instance_snapshot_arn(snapshot_id)
+    account_id = util.get_aws_account_id()
+    bucket_name = constants.RDS_SNAPSHOTS_BUCKET_NAME_PREFIX + account_id
+    try:
+        response = rds.start_export_task(
+            ExportTaskIdentifier=export_id,
+            SourceArn=snapshot_arn,
+            S3BucketName=bucket_name,
```

Comment:
Why don't se use the same bucket name that was provided by the user in https://github.com/intuit/Trapheus/blob/master/README.md  -> Instructions -> Setup -> point 2.  ?

Reply:
We can use that too, but that bucket is meant for Trapheus Cloudformation deployment files right. So I did not want to store the snapshots there. What do you guys think @namitad and @stationeros

Reply:
Also not sure how we can get that bucket name in our lambdas, unless the user passes it in params, which I don't think should be done

Reply:
s3 bucket name needs to be a input as its not part of the state machine while executing sam deploy